xpano/gui/pano_gui.cc

```diff
@@ -343,6 +343,14 @@ Action PanoGui::ResolveFutures() {
     if (!result.pano) {
       info_message_ = fmt::format("Failed to stitch pano {}", result.pano_id);
       spdlog::info(info_message_);
+      switch(result.export_code) {
+        case 1: tooltip_message_ = "Error: Need more images";
```

Comment:
Please use the actual enum values from opencv here [docs](https://docs.opencv.org/4.x/d2/d8d/classcv_1_1Stitcher.html#a507409ce9435dd89857469d12ec06b45), so that this doesn't break if opencv changes.

Also add a default clause, this is good practice if the enum changes in the future.